src/app/(dashboard)/a/settings/component/setting-card.tsx

```diff
@@ -0,0 +1,112 @@
+'use client'
+import { Button } from "@/components/ui/button"
+import { Card, CardHeader, CardTitle } from "@/components/ui/card"
+import { Label } from "@/components/ui/label"
+import Link from "next/link"
+import { useForm } from "react-hook-form"
+
+
+export default function SettingsCard() {
+  const {
+    register,
+    handleSubmit
+  } = useForm({})
+
+  const onSubmit = async (formData: any) => {
+    try {
+      formData.port = parseInt(formData.port, 10);
+      formData.secure = parseBoolean(formData.secure);
+      const response = await fetch('/api/settings', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        console.log('Nice!!!!!!!!!!');
+      }
+    } catch (error) {
+      alert('Errorrrrr');
+    }
+  };
+  const parseBoolean = (value: string) => {
+    if (value === 'true' || value === '1' || value === 'yes') {
+      return true;
+    } else if (value === 'false' || value === '0' || value === 'no') {
+      return false;
+    }
+    // Default to null or handle invalid inputs as needed
+    return null;
+  };
+
+  return (
+    <div className="h-screen w-full flex flex-col lg:flex-row">
+      <nav className="w-full lg:w-64 bg-black text-white p-5">
+        <ul className="space-y-2">
+          <li>
+            <Link className="flex items-center gap-3 py-2 mt-4" href="#">
+              <IconSettings className="h-5 w-5" />
+              <span>SMTP Setting</span>
+            </Link>
+          </li>
+        </ul>
+      </nav>
+      <Card className="w-screen ">
+        <CardHeader>
+          <CardTitle>SMTP Setting</CardTitle>
+        </CardHeader>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="space-y-4">
+            <div className="space-y-2">
+              <Label htmlFor="host" className=" ml-10 mr-10">Host</Label>
+              <input className="p-2 ml-2 border border-black rounded w " type="text" placeholder="smtp.gmail.com" {...register("host", { required: true })} />
+            </div>
+            <div className="space-y-2">
+              <Label htmlFor="port" className="ml-10 mr-11">Port</Label>
+              <input className="p-2 ml-2 border border-black rounded" type="text" placeholder="587" {...register("port", { required: true })} />
+            </div>
+            <div className="space-y-2">
+              <Label htmlFor="port" className="ml-10 mr-6">Secure</Label>
+              <input className="p-2 ml-2 border border-black rounded" type="text" placeholder="false" {...register("secure", { required: true })} />
```

Comment:
Make this a dropdown selection instead of text input so you don't have to parse the value.